### PR TITLE
Amazoncloud

### PR DESCRIPTION
**Pull request by shitalmule04**
##### SUMMARY

All bugs gets fixed at _ec2_vpc file

##### ISSUE TYPE

 - Bugfix Pull Request
 
##### COMPONENT NAME

amazon/_ec2_vpc
##### ANSIBLE VERSION

```
2.5
```

##### ADDITIONAL INFORMATION

```

```

### CODE REVIEW

File: lib/ansible/cli/config.py [hunk 1]
```diff
@@ -1,3 +1,20 @@
...
+# Ansible is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with Ansible.  If not, see <http://www.gnu.org/licenses/>.
+#
+# ansible-vault is a script that encrypts/decrypts YAML files. See
+# http://docs.ansible.com/playbooks_vault.html for more details.
```
Comment (bcoca) [inline lib/ansible/cli/config.py:17]: invalid change, the correct license header is below

**Review by bcoca — CHANGES_REQUESTED** [review]

File: lib/ansible/cli/config.py (hunk 1 above)
Comment (shitalmule04) [reply lib/ansible/cli/config.py:17]: @bcoca  once checks is being complete. I will modify.

Comment (gundalow) [reply lib/ansible/cli/config.py:17]: Still needs fixing

**Comment by gundalow** [comment]
As agreed in 2017-12-07 Core meeting, we've done bulk pep8 fixes across all modules in `devel`.

This was done in https://github.com/ansible/ansible/pull/33682 and https://github.com/ansible/ansible/pull/33690 which have been merged into `devel`. because of this you will need to rebase your PR. Thanks in advance.

**Comment by gundalow** [comment]
Please rebase so this only contains the bug fix

**Pull request closed without merging**